weblab_cg/tests/test_chaste_cg.py

```diff
@@ -0,0 +1,528 @@
+#
+# Tests the basics of weblab_cg
+#
+import cellmlmanip
+import logging
+import os
+import re
+import weblab_cg as cg
+import pytest
+import sympy
+from sympy import SympifyError
+import pyparsing
+
+# Show more logging output
+logging.getLogger().setLevel(logging.DEBUG)
+# TODO: Better docstrings
+
+
+class TestChasteCG(object):
+    _COMMENTS_REGEX = re.compile(r'(//.*\n)')
+    _NUM_REGEX = re.compile(r'(?:0|[1-9]\d*)(?:\.\d*)?(?:[eE][+\-]?\d+)?')
+    _FLOAT_PRECISION = 11
+
+    def _make_dirs(self, output_path):
```

Comment:
I don't think we need private methods in the tests, do we? Can these just be public?

Reply:
They can but I don't really see the benefit.